pages/common/aws-pricing.md

```diff
@@ -0,0 +1,24 @@
+# aws pricing
+
+> Centralized and convenient way to programmatically query Amazon Web Services for services, products, and pricing information.
```

Comment:
Perhaps this better
```suggestion
> Query services, products, and pricing information from Amazon Web Services.
```

Reply:
Much better. Mine was the official description from AWS docs.